source/client/process_impl.cc

```diff
@@ -455,6 +477,28 @@ bool ProcessImpl::runInternal(OutputCollector& collector, const std::vector<UriP
 
     Runtime::LoaderSingleton::get().initialize(*cluster_manager_);
 
+    std::list<std::unique_ptr<Envoy::Stats::Sink>> stats_sinks;
```

Comment:
runInternal is getting a bit too long to follow. Can we try to think of a different structure where the new (or some part of the old) code gets moved into a well named function (or functions) in the anonymous namespace?

That way this gets to keep the high level business logic of the run and the reader can absorb that before looking at the implementation details.

Reply:
Moved the new code of set up sinks to a member function `setupStatsSinks()`.